File: Backend/Backend/Auth/Models/AuthenticateResponse.cs
using Model.SchoolModel.DTO;
using Model.UserModel.DTO;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Backend.Auth.Models
{
    public class AuthenticateResponse
    {
        public Guid Id { get; set; }
        public string Login { get; set; }
        public RolaDTO Rola { get; set; }
        public SchoolDTO School { get; set; }
        public string Token { get; set; }


        public AuthenticateResponse(UserDTO user, string token)
        {
            Id = user.Id;
            Login = user.Login;
            Rola = user.Rola;
            School = user.School;
            Token = token;
        }
    }
}

File: Front/src/app/_models/authenticate_response.ts
import { Guid } from "guid-typescript";
import { Rola } from "./rola";
import { School } from "./school";

export class AuthenticateResponse {
    public id!: Guid;
    public login!: string;
    public rola!: Rola;
    public school!:School;
    public token!: string;
  }